src/App.js

```diff
@@ -11,11 +11,14 @@ import StartPage from './Pages/StartPage/StartPage'
 import GuestPage from './Pages/GuestPage/GuestPage'
 import PacklistForm from './Pages/PacklistForm/PacklistForm'
 
+
 export default function App() {
+
   const user = useContext(UserContext);
-  console.log('app user', user, !!user);
+  console.log('user', user)
```

Comment:
It's not useful for the user to have logs like this. It's a good practice to avoid logging data for development. 

```suggestion
```